FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/BottomNavigationActivity.kt

```diff
@@ -23,17 +23,23 @@ class BottomNavigationActivity : DemoActivity() {
         super.onCreate(savedInstanceState)
 
         toggle_label_button.setOnClickListener {
+            var labelsState: String = ""
             // You can also achieve unlabeled items via @style/Widget.FluentUI.BottomNavigation.Unlabeled
-            bottom_navigation.labelVisibilityMode =
-                if (bottom_navigation.labelVisibilityMode == LabelVisibilityMode.LABEL_VISIBILITY_UNLABELED)
-                    LabelVisibilityMode.LABEL_VISIBILITY_LABELED
-                else
-                    LabelVisibilityMode.LABEL_VISIBILITY_UNLABELED
+            if (bottom_navigation.labelVisibilityMode == LabelVisibilityMode.LABEL_VISIBILITY_UNLABELED){
+                bottom_navigation.labelVisibilityMode = LabelVisibilityMode.LABEL_VISIBILITY_LABELED
+                labelsState = "ON"
+            }
+            else {
+                bottom_navigation.labelVisibilityMode = LabelVisibilityMode.LABEL_VISIBILITY_UNLABELED
+                labelsState = "OFF"
```

Comment:
All caps for announcement can pronounce  'O' 'F' 'F' rather than Off , can you verify this in <10 android as well or just prefer word case 'Off' & 'on'

Reply:
It announced correctly on Android 9.0, but changed them to sentence case as mentioned here.